packages/network-clients/src/config.ts

```diff
@@ -28,26 +28,26 @@ export const NETWORK_CONFIGS: Record<SQNetworks, NetworkConfig> = {
     gql: {
       // TODO: change to kepler-prod endpoint
       [GqlEndpoint.Explorer]:
-        process.env.KEPLER_SUBQL ?? 'https://api.subquery.network/sq/subquery/kepler-testnet-subql-project',
+        process.env.KEPLER_SUBQL ?? 'https://api.subquery.network/sq/subquery/kepler-subquery-project-mumbai',
```

Comment:
We should improve this config file in future
Defenetify can lift the vraibales out of the assignment expression

Reply:
Remove all the default values and config in the CI env